src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,18 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.lenght === 0 ? (
```

Comment:
There is a typo in the condition `comments.lenght === 0`. It should be `comments.length === 0`. This is an error related to the task requirements, as it could cause the component to not display the 'No comments yet' message correctly.